pkgs/applications/office/paperwork/src.nix

```
@@ -1,14 +1,40 @@
-{ fetchFromGitLab }:
+{
+  fetchFromGitLab,
+  stdenv,
+  fetchpatch,
+}:
 rec {
   version = "2.2.5";
-  src = fetchFromGitLab {
-    domain = "gitlab.gnome.org";
-    repo = "paperwork";
-    group = "World";
-    owner = "OpenPaperwork";
-    rev = version;
-    sha256 = "sha256-PRh0ohmPLwpM76qYfbExFqq4OK6Hm0fbdzrjXungSoY=";
-  };
+  src =
+    let
+      actual_src = fetchFromGitLab {
+        domain = "gitlab.gnome.org";
+        repo = "paperwork";
+        group = "World";
+        owner = "OpenPaperwork";
+        rev = version;
+        sha256 = "sha256-PRh0ohmPLwpM76qYfbExFqq4OK6Hm0fbdzrjXungSoY=";
+      };
+    in
+    stdenv.mkDerivation {
+      pname = "paperwork-patched-src";
+      inherit version;
+      src = actual_src;
+
+      patches = [
+        # fix installing translations
+        # remove on next release
+        (fetchpatch {
+          url = "https://gitlab.gnome.org/World/OpenPaperwork/paperwork/-/commit/b8e2633ace0f3d9d57e68c27db8f594b8a5ddd7e.patch";
+          hash = "sha256-VUT86kF0ZHLGK457ZrrIBMeiZqg/rPRpbkBA/ua9rU8=";
+        })
+      ];
+
+      installPhase = ''
+        cp -r . $out
+      '';
+    };
```

```suggestion
    srcOnly {
      pname = "paperwork-patched-src";
      inherit version;
      src = actual_src;

      patches = [
        # fix installing translations
        # remove on next release
        (fetchpatch {
          url = "https://gitlab.gnome.org/World/OpenPaperwork/paperwork/-/commit/b8e2633ace0f3d9d57e68c27db8f594b8a5ddd7e.patch";
          hash = "sha256-VUT86kF0ZHLGK457ZrrIBMeiZqg/rPRpbkBA/ua9rU8=";
        })
      ];
    };
```

... and I'd even go further and inline `actual_src`, since it only is used once.